README.md

```diff
@@ -5,7 +5,7 @@ Downloads archived MBTA real-time data feeds from various sources.
 ## Prerequisites
 
 * Python 3
-  * **Mac/Linux:** can install using [asdf](https://asdf-vm.com/) (`asdf install` in this directory)
+  * **Mac/Linux:** can install using [asdf](https://asdf-vm.com/) (`asdf install` in this directory) and then `pip install --user pipenv`
```

Comment:
suggestion: this doc change should probably be its own commit, it doesn't match the commit title. thank you for updating this, though!

Reply:
https://github.com/mbta/prediction_loc/pull/32